src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/wwwroot/lib/jquery/dist/jquery.min.js

```diff
@@ -1,9 +1,2 @@
-<<<<<<< HEAD
```

Comment:
🤦‍♂️ let's open a bug to fix this in the 2.1.x patches as well.

Reply:
Sure, will do